deploy/k8s/charts/trustification/templates/services/guac/graphql/030-Deployment.yaml

```diff
@@ -66,6 +66,8 @@ spec:
             - name: GUAC_GQL_TLS_KEY_FILE
               value: /etc/tls/tls.key
             {{- end }}
+            - name: GUAC_PROMETHEUS_ADDR
```

Comment:
This should only be enabled if the user requests metrics. Take a look at the `_metrics.tpl` file.

Reply:
@ctron as far as I can see `trustification.application.metrics.enabled` only affect pod labels, which is also the case for this pod. 
This env only adjust the usage of port `9010` (from the default `9091`) as it's used for other services. All other services have as well have a `:9010/metrics` enabled by default.

Reply:
For all others this is controlled using the `INFRASTRUCTURE_*` env-vars. I don't know what's way to control this one. But I think that it should also be put into some "include" so that we can control that later on … vs spreading hardcoded options throughout the charts.

Reply:
I pushed another commit in attempt to make infrastructure port configurable and use that value. But I didn't have time to finish it. Creating schemas seems more complicated than it should be. Maybe we can add some docs on how to maintain them for others as well. Anyhow, I've put this in draft for now and will pick it up when I'm back.